src/SourceLocator/Type/Composer/Psr/Psr0Mapping.php

```diff
@@ -54,7 +54,7 @@ public function resolvePossibleFilePaths(Identifier $identifier): array
         foreach ($this->mappings as $prefix => $paths) {
             if (strpos($className, $prefix) === 0) {
                 return array_map(
-                    static fn (string $path): string => rtrim($path, '/') . '/' . str_replace(['\\', '_'], '/', $className) . '.php',
```

Comment:
The ending `/` in the path is valid in PSR-0 and PSR-4 mappings - should probably be tested.

Reply:
It's already removed in https://github.com/Roave/BetterReflection/blob/5.1.x/src/SourceLocator/Type/Composer/Psr/Psr0Mapping.php#L37 and tested as well.